app/controllers/api/integrations_controller.rb

```diff
@@ -87,6 +87,7 @@ def promote_to_production
   end
 
   def show
+    @apiap = apiap?
```

Comment:
This is to be able to use it from the views... so imo it should be a helper instead :thinking: 
but i know it is a temporary thing bcz soon everything will be apiap and we will remove this, so ok :+1: